### PR TITLE
[zh-cn]: update the translation of Document `lastModified` property

### PR DESCRIPTION
**Pull request by T34-active**
### Description
https://developer.mozilla.org/en-US/docs/Web/API/Document/lastModified
### Related issues and pull requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/document/lastmodified/index.md
* Last commit: https://github.com/mdn/content/commit/58ad1df59f2ffb9ecab4e27fe1bdf1eb5a55f89b
* Document last modified: 2024-07-24T01:12:06.000Z

### CODE REVIEW

File: files/zh-cn/web/api/document/lastmodified/index.md [hunk 1]
```diff
@@ -1,31 +1,92 @@
...
+// 返回：Tuesday, December 16, 2017 11:09:42
 ```
 
-### 例子
+### 将 lastModified 转换为日期对象
 
-```plain
-dump(document.lastModified);
-// 返回：Tuesday, July 10, 2001 10:19:42
+此示例将 `lastModified` 转换为 {{jsxref(“Date”)}} 对象。
```
Comment (yin1999) [inline files/zh-cn/web/api/document/lastmodified/index.md:29]:
```suggestion
此示例将 `lastModified` 转换为 {{jsxref("Date")}} 对象。
```

File: files/zh-cn/web/api/document/lastmodified/index.md [hunk 2]
```diff
@@ -1,31 +1,92 @@
...
+```
+
+## 备注
+
+请注意，作为字符串的 `lastModified` 无法*轻松*用于比较文档的修改日期。
+
+下面是一个页面切换时显示提示消息的示例（参见：[JavaScript cookies API](/zh-CN/docs/Web/API/Document/cookie)）：
+
+```js
+// 与 'timestamp' 中的 'last_modif=timestamp' 匹配
```
Comment (jasonren0403) [inline files/zh-cn/web/api/document/lastmodified/index.md:50]:
```suggestion
// 在 'last_modif=timestamp' 中匹配 'timestamp'
```

File: files/zh-cn/web/api/document/lastmodified/index.md [hunk 3]
```diff
@@ -1,31 +1,92 @@
...
+    location.pathname
+  }`;
+
+  if (isFinite(lastVisit)) {
+    alert("此页面已更改！");
+  }
+}
+```
+
+如果你想知道某个*外部*页面是否发生了变化，可以使用 {{HTTPMethod("HEAD")}} 请求并通过 {{domxref("Window/fetch", "fetch()")}} API 发起请求，并检查 {{HTTPHeader("Last-Modified")}} 响应标头信息。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/document/lastmodified/index.md:84]:
```suggestion
如果你想知道某个*外部*页面是否发生了变化，可以通过 {{domxref("Window/fetch", "fetch()")}} API 发起 {{HTTPMethod("HEAD")}} 请求，并检查 {{HTTPHeader("Last-Modified")}} 响应标头信息。
```

File: files/zh-cn/web/api/document/lastmodified/index.md [hunk 4]
```diff
@@ -1,31 +1,92 @@
...
+
+此示例会提示 `lastModified` 的值。
+
+```js
+alert(document.lastModified);
+// 返回：Tuesday, December 16, 2017 11:09:42
 ```
 
-### 例子
+### 将 lastModified 转换为日期对象
```
Comment (jasonren0403) [inline files/zh-cn/web/api/document/lastmodified/index.md:27]:
```suggestion
### 将 lastModified 转换为 Date 对象
```

File: files/zh-cn/web/api/document/lastmodified/index.md [hunk 5]
```diff
@@ -1,31 +1,92 @@
...
+此示例将 `lastModified` 转换为 {{jsxref("Date")}} 对象。
+
+```js
+let oLastModif = new Date(document.lastModified);
 ```
 
-### 备注
+### 将 lastModified 转换成毫秒
+
+此示例将 `lastModified` 转换为自 1970 年 1 月 1 日 00:00:00（本地时间）以来的毫秒数。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/document/lastmodified/index.md:37]:
```suggestion
此示例将 `lastModified` 转换为自本地时间 1970 年 1 月 1 日 00:00:00 以来的毫秒数。
```

**Review by jasonren0403 — COMMENTED** [review]
一些建议

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**